src/BadgeSelectItem/BadgeSelectItem.js

```diff
@@ -14,13 +14,15 @@ const BadgeSelectItem = props => {
     highlighted,
     disabled,
     className,
+    size,
+    suffix,
   } = props;
 
   const prefix = <div className={st(classes.marker, { skin })} />;
 
   return (
     <ListItemSelect
-      className={className}
+      className={st(classes.root, {}, className)}
```

Comment:
```suggestion
      className={st(classes.root, className)}
```

With stylable 3 no need for an empty object.